script/Ciel/Day 7/07_01_CIEL07_2_1A.txt

```diff
@@ -27,21 +27,22 @@ I get the feeling she's still condemning me for my conversation with Noel this m
 [sha:2eba0d00a2103e4911ea29b2e0f36e64d407dafd]{
 -- Page 174, Offset 15217.
 -- 「そ、そんなコトは訊いてないし気にしてませんけど。
-"T-that wasn't what I was asking about, and I wasn't worried about it either.
+"T-That isn't what I was asking about, and it's not like I'd care if you did.
 }
 [sha:0c32043756a9140c3c19841305cdf3b6cd46835f]{
 -- Page 174, Offset 15218.
 -- ……そうですか。あれはノエル先生の、いつもの暴走片道切符なんですね。まあそんなコトはどうでもいいので早く座ってください。立っていられると落ち着かないです」
 // 暴走片道切符 is so weird here, I don't even know where to begin. It sort of makes sense that its signifying something about Noel having poor impulse control (especially considering how crazy we see her get later in the route), but the phrasing is so odd I don't really know what to translate it as.
 // 落ち着かない here (translated as 'bothered' is reflected a few lines down at Offset 15226.
-But I see... So that was just part of Noel-sensei's usual one-way trip to going out of control.
+But I see...
+ I guess that was just typical Noel-sensei behavior――she's a lost cause the minute she flies off the handle.
```

Comment:
I don't really know how I'd tl this sentence but I think that the implication that Noel is only a 'lost cause' once she goes out of control shouldn't be there (because she was a lost cause from the start).

Reply:
disagree with how you're reading it Yusuke. I think what Ross wrote is saying is that it's a lost cause to try to reason with her once she's gone. Her being a lost cause wholesale and this are just phrased the same way because english.

Maybe the fact that's not clear is a sign that it should be reworded though. "It's a lost cause getting through to her the minute she flies off the handle." or something

Reply:
I do agree that the 'lost cause' phrasing is a bit off, not just because of how its seems interpretation-wise, but it reads kinda weird too.
'Once she flies off the handle, there's no stopping her' might be preferrable.